packages/portfolio-contract/src/resolver/resolver.exo.ts

```diff
@@ -94,10 +126,52 @@ export const prepareResolverKit = (
             key,
             harden({ destinationAddress, amountValue, vowKit }),
           );
+          const [chain, chainId, ethAddress] = destinationAddress.split(':');
+          this.facets.vstorage.insertPendingTransaction(
+            amountValue,
+            `${chain}:${chainId}`,
+            ethAddress as `0x${string}`,
+          );
```

Comment:
why split these up? the orchestration API prefers `AccountId`, I think.

**design** question

@rabi-siddique I gather you asked for it to be split up? why?